src/store/user/user-context.js

```diff
@@ -10,6 +11,18 @@ export const UserContextProvider = ({ children }) => {
   const [showMemberRoleUpdateModal, setShowMemberRoleUpdateModal] =
     useState(false);
   const [selectedMember, setSelectedMember] = useState(null);
+  const [apiCalledToVerifySuperUser, setApiCalledToVerifySuperUser] =
```

Comment:
Could we please use simpler names here, as described here https://github.com/Real-Dev-Squad/website-members/pull/410/files#r952055322?

Reply:
yeah right. Updated the variable names.